src/mesh/generated/meshtastic/telemetry.pb.h

```diff
@@ -57,7 +57,9 @@ typedef enum _meshtastic_TelemetrySensorType {
     /* Lite On LTR-390UV-01 UV Light Sensor */
     meshtastic_TelemetrySensorType_LTR390UV = 21,
     /* AMS TSL25911FN RGB Light Sensor */
-    meshtastic_TelemetrySensorType_TSL25911FN = 22
+    meshtastic_TelemetrySensorType_TSL25911FN = 22,
+    /* AHT10/20 Temperature Sensor */
+    meshtastic_TelemetrySensorType_AHT10 = 23
```

Comment:
need add AHT10 to protobuf like in https://github.com/meshtastic/protobufs/pull/499/files

Reply:
OK, I don't understand in what manner: protobuf folder in my branch must be  included in branch an not in  submodule ?

Reply:
I first we need add  AHT10 support to protobuf https://github.com/meshtastic/protobufs/pull/507/files and when use latest submodule  to use in this MR

Reply:
> I first we need add AHT10 support to protobuf https://github.com/meshtastic/protobufs/pull/507/files and when use latest submodule to use in this MR

So i need to do a pull request in protobuf first ? Can't you enter it directly?

Reply:
already done my me -   https://github.com/meshtastic/protobufs/pull/507/files

Reply:
@fzellini can you add latest  protobuf submodule  to merge request?  https://github.com/meshtastic/protobufs/pull/507  merged

Reply:
Just sync fzellini:trackerd brach with master (protobuf alredy merged https://github.com/meshtastic/firmware/pull/4003/files)